package-lock.json

```diff
@@ -2205,9 +2205,10 @@
       }
     },
     "node_modules/@empathyco/x-adapter": {
-      "version": "8.0.3-alpha.1",
-      "resolved": "https://registry.npmjs.org/@empathyco/x-adapter/-/x-adapter-8.0.3-alpha.1.tgz",
-      "integrity": "sha512-FzQ17CbUvN6p5zLYQyMT9O2JzE0SJQ53wDHqUNFXQykfo1IUBmRMCiJo8Xm7vJpQQB9/de4nEjf5iQxwVeW8aA==",
+      "version": "8.1.0-alpha.0",
```

Comment:
Review you npm configuration, because we can't use nexus here

Reply:
Done